src/templates/partners/omnibus.html.tt

```diff
@@ -0,0 +1,49 @@
+<div class="container">
+    <div class="static_full">
+        <h1>[% l('Omnibus account') %]</h1>
+        <p>[% l('Are you a licensed and regulated broker with a large database of active traders? Manage your database more efficiently by consolidating all your client accounts under an omnibus master account. This allows you to keep their individual identities private at the same time.') %]</p>
+    </div>
+</div>
+
+[% INCLUDE global/separator_line.html.tt class='gr-parent gr-padding-20' invisible=1 %]
+
+<div class="fill-bg-color center-text gr-padding-20">
+    <h2 class="gr-padding-20 gr-child">[% l('Key features') %]</h2>
+    <div class="container gr-row">
+        [% BLOCK box %]
+            <div class="gr-4 gr-12-p gr-12-m gr-parent">
+                <div class="box gr-gutter gr-padding-10">
+                    <img class="responsive gr-padding-20" src="[% request.url_for('images/pages/omnibus/' _ image _ '.svg') %]" />
+                    <strong>[% header %]</strong>
+                    <p>[% text %]</p>
+                </div>
+            </div>
+        [% END %]
+        [% INCLUDE box image='trade'    header=l('More efficient') text=l('Trade on behalf of your clients through one account.') %]
+        [% INCLUDE box image='low-fees' header=l('Lower fees')     text=l('Reduce fees associated with deposits and withdrawals.') %]
+        [% INCLUDE box image='privacy'  header=l('Privacy')        text=l('Maintain complete anonymity of your clients.') %]
+    </div>
+</div>
+
+<div class="container gr-padding-30 gr-child">
+    <h2 class="center-text">[% l('How it works') %]</h2>
+    [% INCLUDE global/separator_line.html.tt class='gr-parent' invisible=1 %]
+    [% BLOCK steps %]
+        <div class="steps gr-12 gr-12-p gr-12-m gr-parent">
+            <p class="center-text">[% text %]</p>
+            [% IF has_arrow -%]
+                <img class="responsive gr-padding-20" src="[% request.url_for('images/pages/omnibus/arrow-down.svg') %]" />
+            [%- END %]
+        </div>
+    [% END %]
+    [% INCLUDE steps has_arrow=1 text=l('Create your own white-label application with the omnibus facility, plus all the features supported by our extensive API library. We won’t know who your clients are and vice versa') %]
+    [% INCLUDE steps has_arrow=1 text=l('Create sub-accounts to record transactions you make on behalf of your clients') %]
+    [% INCLUDE steps has_arrow=1 text=l('Customise individual access for clients who wish to trade for themselves') %]
+    [% INCLUDE steps has_arrow=1 text=l('Transfer funds between your omnibus master account and sub-accounts') %]
+    [% INCLUDE steps text=l('All deposits and withdrawals are made through your omnibus master account') %]
+</div>
+
+<div class="container">
+    <p class="center-text">[% l('Interested in our omnibus account facility? [_1].', '<a href="mailto:marketing@binary.com">Contact us now and apply as an omnibus broker</a>') %]</p>
```

Comment:
i think we need to add 'Contact us now and apply as an omnibus broker' to translation